crates/rust-client/src/accounts.rs

```diff
@@ -75,17 +75,23 @@ impl<R: FeltRng> Client<R> {
         Ok(account_and_seed)
     }
 
-    /// Saves in the store the [Account] corresponding to `account_data`.
+    /// Saves in the store the [Account] corresponding to `account_data`. If the account is already
```

Comment:
This is from before, but I'd change the following:
```suggestion
    /// Saves the [Account] contained in `account_data` in the store. If the account is already
```